<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="5" passed="2" failed="3" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2024-02-21T16:15:45 IST" name="Positive Tests Suite" finished-at="2024-02-21T16:15:58 IST" duration-ms="12980">
    <groups>
    </groups>
    <test started-at="2024-02-21T16:15:45 IST" name="Positive Tests" finished-at="2024-02-21T16:15:58 IST" duration-ms="12980">
      <class name="com.herokuapp.restfulbookerassignment.DeleteBookingTests">
        <test-method is-config="true" signature="setUp()[pri:0, instance:com.herokuapp.restfulbookerassignment.DeleteBookingTests@41e36e46]" started-at="2024-02-21T16:15:55 IST" name="setUp" finished-at="2024-02-21T16:15:55 IST" duration-ms="1" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="partialUpdateBookingTest()[pri:0, instance:com.herokuapp.restfulbookerassignment.DeleteBookingTests@41e36e46]" started-at="2024-02-21T16:15:55 IST" name="partialUpdateBookingTest" finished-at="2024-02-21T16:15:58 IST" duration-ms="3704" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- partialUpdateBookingTest -->
      </class> <!-- com.herokuapp.restfulbookerassignment.DeleteBookingTests -->
      <class name="com.herokuapp.restfulbookerassignment.UpdateBookingTests">
        <test-method is-config="true" signature="setUp()[pri:0, instance:com.herokuapp.restfulbookerassignment.UpdateBookingTests@76508ed1]" started-at="2024-02-21T16:15:53 IST" name="setUp" finished-at="2024-02-21T16:15:53 IST" duration-ms="1" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="createBookingTest()[pri:0, instance:com.herokuapp.restfulbookerassignment.UpdateBookingTests@76508ed1]" started-at="2024-02-21T16:15:53 IST" name="createBookingTest" finished-at="2024-02-21T16:15:55 IST" duration-ms="1723" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- createBookingTest -->
      </class> <!-- com.herokuapp.restfulbookerassignment.UpdateBookingTests -->
      <class name="com.herokuapp.restfulbookerassignment.GetBookingTests">
        <test-method is-config="true" signature="setUp()[pri:0, instance:com.herokuapp.restfulbookerassignment.GetBookingTests@4ac3c60d]" started-at="2024-02-21T16:15:49 IST" name="setUp" finished-at="2024-02-21T16:15:49 IST" duration-ms="1" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="getBookingTest()[pri:0, instance:com.herokuapp.restfulbookerassignment.GetBookingTests@4ac3c60d]" started-at="2024-02-21T16:15:49 IST" name="getBookingTest" finished-at="2024-02-21T16:15:51 IST" duration-ms="1896" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[The following asserts failed:
firstname in response is not expected expected [Eric] but found [Vijay],
lastname in response is not expected expected [Brown] but found [Singhal],
totalprice in response is not expected expected [140] but found [543],
depositpaid should be true but it is not expected [true] but found [false],
checkin response is not expected expected [2019-05-29] but found [2024-09-08],
checkin response is not expected expected [2020-03-04] but found [2024-09-10],
additional needs in repsonse is not expected expected [Breakfast] but found [dryer]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: The following asserts failed:
firstname in response is not expected expected [Eric] but found [Vijay],
lastname in response is not expected expected [Brown] but found [Singhal],
totalprice in response is not expected expected [140] but found [543],
depositpaid should be true but it is not expected [true] but found [false],
checkin response is not expected expected [2019-05-29] but found [2024-09-08],
checkin response is not expected expected [2020-03-04] but found [2024-09-10],
additional needs in repsonse is not expected expected [Breakfast] but found [dryer]
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:47)
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:31)
at com.herokuapp.restfulbookerassignment.GetBookingTests.getBookingTest(GetBookingTests.java:47)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getBookingTest -->
      </class> <!-- com.herokuapp.restfulbookerassignment.GetBookingTests -->
      <class name="com.herokuapp.restfulbookerassignment.PartialUpdateBookingTests">
        <test-method is-config="true" signature="setUp()[pri:0, instance:com.herokuapp.restfulbookerassignment.PartialUpdateBookingTests@4facf68f]" started-at="2024-02-21T16:15:51 IST" name="setUp" finished-at="2024-02-21T16:15:51 IST" duration-ms="1" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="partialUpdateBookingTest()[pri:0, instance:com.herokuapp.restfulbookerassignment.PartialUpdateBookingTests@4facf68f]" started-at="2024-02-21T16:15:51 IST" name="partialUpdateBookingTest" finished-at="2024-02-21T16:15:53 IST" duration-ms="2171" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[The following asserts failed:
lastname in response is not expected expected [Shyshkin] but found [Singhal],
totalprice in response is not expected expected [150] but found [543],
additional needs in repsonse is not expected expected [breakfast] but found [dryer]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: The following asserts failed:
lastname in response is not expected expected [Shyshkin] but found [Singhal],
totalprice in response is not expected expected [150] but found [543],
additional needs in repsonse is not expected expected [breakfast] but found [dryer]
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:47)
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:31)
at com.herokuapp.restfulbookerassignment.PartialUpdateBookingTests.partialUpdateBookingTest(PartialUpdateBookingTests.java:60)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- partialUpdateBookingTest -->
      </class> <!-- com.herokuapp.restfulbookerassignment.PartialUpdateBookingTests -->
      <class name="com.herokuapp.restfulbookerassignment.CreateBookingTests">
        <test-method is-config="true" signature="setUp()[pri:0, instance:com.herokuapp.restfulbookerassignment.CreateBookingTests@be64738]" started-at="2024-02-21T16:15:45 IST" name="setUp" finished-at="2024-02-21T16:15:46 IST" duration-ms="333" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="createBookingTest()[pri:0, instance:com.herokuapp.restfulbookerassignment.CreateBookingTests@be64738]" started-at="2024-02-21T16:15:46 IST" name="createBookingTest" finished-at="2024-02-21T16:15:49 IST" duration-ms="3119" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[The following asserts failed:
firstname in response is not expected expected [Anuj] but found [Vijay],
lastname in response is not expected expected [Kumar] but found [Singhal],
totalprice in response is not expected expected [150] but found [543],
checkin response is not expected expected [2020-03-25] but found [2024-09-08],
checkin response is not expected expected [2020-03-27] but found [2024-09-10],
additional needs in repsonse is not expected expected [Breakfast] but found [dryer]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: The following asserts failed:
firstname in response is not expected expected [Anuj] but found [Vijay],
lastname in response is not expected expected [Kumar] but found [Singhal],
totalprice in response is not expected expected [150] but found [543],
checkin response is not expected expected [2020-03-25] but found [2024-09-08],
checkin response is not expected expected [2020-03-27] but found [2024-09-10],
additional needs in repsonse is not expected expected [Breakfast] but found [dryer]
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:47)
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:31)
at com.herokuapp.restfulbookerassignment.CreateBookingTests.createBookingTest(CreateBookingTests.java:47)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- createBookingTest -->
      </class> <!-- com.herokuapp.restfulbookerassignment.CreateBookingTests -->
    </test> <!-- Positive Tests -->
  </suite> <!-- Positive Tests Suite -->
</testng-results>
